x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -125,36 +133,55 @@ const DisplayList = memo(function DisplayList({
   );
 });
 
-export const ProcessEventList = memo(function ProcessEventList({
+export function NodeEventsOfType({ nodeID, eventType }: { nodeID: string; eventType: string }) {
+  const processEvent = useSelector((state: ResolverState) =>
+    selectors.processEventForID(state)(nodeID)
+  );
+  const relatedEventsStats = useSelector((state: ResolverState) =>
+    selectors.relatedEventsStats(state)(nodeID)
+  );
+
+  return (
+    <StyledPanel>
+      <NodeEventList
+        processEvent={processEvent}
+        eventType={eventType}
+        relatedStats={relatedEventsStats}
+      />
+    </StyledPanel>
+  );
+}
+
+const NodeEventList = memo(function ProcessEventList({
   processEvent,
   eventType,
   relatedStats,
 }: {
-  processEvent: ResolverEvent;
+  processEvent: ResolverEvent | null;
   eventType: string;
-  relatedStats: ResolverNodeStats;
+  relatedStats: ResolverNodeStats | undefined;
 }) {
   const processName = processEvent && event.eventName(processEvent);
-  const processEntityId = event.entityId(processEvent);
-  const totalCount = Object.values(relatedStats.events.byCategory).reduce(
-    (sum, val) => sum + val,
-    0
+  const processEntityId = processEvent ? event.entityId(processEvent) : '';
+  const nodesHref = useSelector((state: ResolverState) =>
+    selectors.relativeHref(state)({ panelView: 'nodes' })
   );
+
+  const nodesLinkNavProps = useNavigateOrReplace({
+    search: nodesHref,
+  });
+  const totalCount = relatedStats
+    ? Object.values(relatedStats.events.byCategory).reduce((sum, val) => sum + val, 0)
+    : 0;
   const eventsString = i18n.translate(
     'xpack.securitySolution.endpoint.resolver.panel.processEventListByType.events',
     {
       defaultMessage: 'Events',
     }
   );
-  const waitingString = i18n.translate(
-    'xpack.securitySolution.endpoint.resolver.panel.processEventListByType.wait',
-    {
-      defaultMessage: 'Waiting For Events...',
-    }
-  );
 
   const relatedsReadyMap = useSelector(selectors.relatedEventsReady);
-  const relatedsReady = relatedsReadyMap.get(processEntityId);
+  const relatedsReady = processEntityId && relatedsReadyMap.get(processEntityId);
```

Comment:
This check probably isn't right since `''` would be falsy while still being a valid entity_id

Reply:
fixed in next PR